Logstash-Configuration/README.md

```diff
@@ -254,7 +254,7 @@ Create Required Directories
 
 Using the query below we can query if we are getting logs with GeoIP information into Azure Sentinel
 
-    // PFSesne GeoIp Traffic
+    // PFSense GeoIp Traffic
```

Comment:
![image](https://user-images.githubusercontent.com/15075388/102724760-d8714280-4309-11eb-98a0-fe8b806e6042.png)

Thanks for the spelling correction, however, please could you correct this to `pfSense`??